src/prefect/engine/executors/base.py

```diff
@@ -24,13 +24,16 @@ def start(self) -> Iterator[None]:
         """
         yield
 
-    def submit(self, fn: Callable, *args: Any, **kwargs: Any) -> Any:
+    def submit(
+        self, fn: Callable, *args: Any, extra_context: dict = None, **kwargs: Any
+    ) -> Any:
         """
         Submit a function to the executor for execution. Returns a future-like object.
 
         Args:
             - fn (Callable): function that is being submitted for execution
             - *args (Any): arguments to be passed to `fn`
+            - extra_context (dict, optional): an optional dictionary of executor-specific keyword arguments
```

Comment:
oop, approved too quickly. Could you update the description for this as well? Something like:

```
an optional dictionary with extra information about the submitted task
```

Reply:
yup you got it

Reply:
Updated